CHANGELOG.adoc

```diff
@@ -0,0 +1,103 @@
+:toc:
+
+= Changelog
+
+== v1.7.1 (2018-13-11)
```

Comment:
As discussed, this could now be for 1.8.0.

Reply:
Done